packages/@dcl/inspector/src/lib/sdk/components.ts

```diff
@@ -86,5 +94,26 @@ export function createEditorComponents(engine: IEngine): EditorComponents {
     )
   })
 
-  return { Selection, Scene, Nodes }
+  const Actions = engine.defineComponent(EditorComponentNames.Actions, {
+    value: Schemas.Array(
+      Schemas.Map({
+        name: Schemas.String,
+        type: Schemas.EnumString<AvailableActions>(AvailableActions, AvailableActions.PLAY_ANIMATION)
+      })
+    )
+  })
+
+  const Triggers = engine.defineComponent(EditorComponentNames.Triggers, {
+    value: Schemas.Array(
+      Schemas.Map({
+        type: Schemas.EnumString<AvailableTriggers>(AvailableTriggers, AvailableTriggers.ON_CLICK),
+        entity: Schemas.Optional(Schemas.Entity),
+        action: Schemas.Optional(
```

Comment:
the `action` field of the trigger should be the `name` of the action rather than the `type` (so it should be a `string` instead of an enum).

This is because an entity could have multiple `play_animation` actions, like a door that has `open` and `close` animations.

Reply:
you're right! updating it